client/blocks/reader-full-post/_content.scss

```diff
@@ -265,6 +265,22 @@
 		border-color: var( --color-neutral-10 );
 	}
 
+	.wp-block-table table {
+		border-collapse: collapse;
+	}
+
+	.wp-block-table td,
+	.wp-block-table th {
+		border: 1px solid #767676;
```

Comment:
Is there a suitable `var` for this colour?

Reply:
Do you use this source code to find these? https://github.com/Automattic/color-studio

Reply:
done.